### DISCUSSION TITLE
Fresh AWS 1.1.6 stack couldn't import content

### DISCUSSION THREAD

**Discussion by sarurururu** [mozilla/hubs#6105] [Hubs Cloud Questions]
**Description**
A fresh Hubs cloud stack (both personal and enterprise versions) set up using the 1.1.6 AWS template with #6049 S3 bucket issue workaround cannot import content (stuck at rotating blue circle)

**To Reproduce**
1. Go to 'Content>Import Content'
2. Specify a .pack/ URL, then Click on 'PREVIEW IMPORT'
3. See error

**Expected behavior**
Showing a preview of the avatar or scene to be imported

**Screenshots**
<img width="869" alt="Screenshot 2023-05-29 114044" src="https://github.com/mozilla/hubs/assets/74574559/688e7e17-6ee1-4995-bc94-752a8d2b8302">

**Hardware** 
 - Device: [Desktop]
 - OS: [Windows 11]
 - Browser: [Firefox/ Chrome/ Edge]

**Additional context**
The same happens after updating the stack using the stock AWS 1.1.6 template (without workarounds). Using the AP-NE-1 (Tokyo) server.

**Comment by msub2** [comment]
Also observed this when attempting to set up Hubs Cloud AWS for the first time today. Attempting to import an avatar pack yields two failed network requests:

- A 404 to `https://<cors-proxy>.<internal domain>/<request URL>`, which if followed shows a GitHub pages 404 error page
- A 403 to `https://<cors-proxy>.<internal domain>/null/api/v1/avatars/undefined`

**Comment by tietoafinlandoy** [comment]
Same here, jamming to the "import content" as described above on fresh default template setup on Ireland server using MacBook Pro M1 (not that it would matter here).

<img width="783" alt="image" src="https://github.com/mozilla/hubs/assets/56432442/cc08eb36-af77-4103-979b-174d7abb1c64">

On browser console errors: Uncaught (in promise) SyntaxError: Unexpected token '<', "<?xml vers"... is not valid JSON

<img width="251" alt="image" src="https://github.com/mozilla/hubs/assets/56432442/438f695e-3c08-4fd1-b6f5-d092c4f235de">

**Reply by tietoafinlandoy** [reply]
@keianhzo with my limited knowledge and skills - this seems to be preventing finalising the deployment-setup as we cannot get past the obligatory setup of avatar and scene. Any links to follow to find a detour?

**Comment by nurse-the-code** [comment]
This bug also appears to be present in `hubs-compose`.
